lib/zlib.js

```diff
@@ -326,15 +332,8 @@ function Zlib(opts, mode) {
     }
   }
 
-  if (opts.strategy) {
-    if (opts.strategy != constants.Z_FILTERED &&
-        opts.strategy != constants.Z_HUFFMAN_ONLY &&
-        opts.strategy != constants.Z_RLE &&
-        opts.strategy != constants.Z_FIXED &&
-        opts.strategy != constants.Z_DEFAULT_STRATEGY) {
-      throw new Error('Invalid strategy: ' + opts.strategy);
-    }
-  }
+  if (opts.strategy && !(strategies.includes(opts.strategy)))
```

Comment:
Happy to remove it if it's bothersome, but the reason I (sometimes) use the parentheses like that are:

* clarity of intention
* helps out people who may not know operator precedence
* force of habit from `!(foo instanceof Buffer)` being the correct way to do things because `!foo instanceof Buffer` means checking that `!foo` is an instance of `Buffer` which is not at all the same as `foo not an instances of Buffer`....